README.md

```diff
@@ -125,18 +142,18 @@ The workflow requires the following parameters to be specified before running:
 * `User Count`: The number of users to pre-seed the Integreatly environment with
 * `Self Signed Certs`: Set to `false` by default. Set to `true` if the target Openshift cluster uses self signed certificates
 
-### Cluster Create
+### 4.5 Cluster Create
 
 Once the tower bootstrapping has been run you can bootstrap the cluster create resources. To create all the resources necessary to run a cluster create you must run the `bootstrap_cluster_create.yml` playbook. The playbook doesn't take any extra variables so the command to run is:
 
 ```bash
-ansible-playbook -i inventories/hosts playbooks/bootstrap_cluster_create.yml
+ansible-playbook -i <path-to-local-credentials-project>/hosts playbooks/bootstrap_cluster_create.yml
```

Comment:
You'll need --ask-vault-pass if you're specifying a creds repo, And you need one for the cluster create.